.github/workflows/autoupdate-prod.yaml

```diff
@@ -64,14 +66,9 @@ jobs:
             ## Manual Review Procedure
 
             1. Review changes in the OpenAPI file (openapi/atlas-api.yaml)
-            2. If we see any breaking changes [without a new resource version](https://github.com/mongodb/atlas-sdk-go/tree/main/tools/releaser#new-resource-version).
```

Comment:
Simplification for the reviewers. We no longer need to understand Versioned API and SDK versioning scheme to review PR.

Reply:
QQ, does this mean we're comfortable assuming all SDK breaking changes will be caught in the generated file (no need to double check)?

Reply:
The key fix here was the bug we had for Major releases that prevented us from getting any breaking changes even in manual form. Since I made this improvement I decided to autogenerate them. 
If we see that autogeneration does have some issues we can always turn it off.

Reply:
Is the manual major release approach still documented somewhere in case we do have some issues with autogeneration?